src/simulation/simulation_sequence.jl

```diff
@@ -192,12 +192,49 @@ function _attach_feedforwards(models::SimulationModels, feedforwards)
     return ff_dict
 end
 
-@doc raw"""
+"""
     SimulationSequence(
-                        models::SimulationModels,
-                        feedforward::Dict{Symbol, <:AbstractAffectFeedforward}
-                        ini_cond_chronology::Dict{Symbol, <:FeedforwardChronology}
-                        )
+        models::SimulationModels,
+        feedforward::Dict{String, Vector{<:AbstractAffectFeedforward}}
+        ini_cond_chronology::InitialConditionChronology
+    )
+
+Construct the simulation sequence between decision and emulation models.
+
+# Arguments
+
+  - `models::SimulationModels`: Vector of decisions and emulation models.
+  - `feedforward = Dict{String, Vector{<:AbstractAffectFeedforward}}()`: Optional dictionary to specify how information
```

Comment:
Is this effort covering feedforwards? If so, this would be a good place to refer to documentation that describes it in detail.

Reply:
agreed. We should check that the FeedForwards have correct docstrings